full-service/src/service/sync.rs

```diff
@@ -110,14 +124,16 @@ impl Drop for SyncThread {
 pub fn sync_all_accounts(
     ledger_db: &LedgerDB,
     conn: Conn,
+    accounts_with_deposits: Arc<Mutex<HashMap<AccountID, bool>>>,
     logger: &Logger,
 ) -> Result<(), SyncError> {
     // Get the current number of blocks in ledger.
     let num_blocks = ledger_db
         .num_blocks()
         .expect("failed getting number of blocks");
     if num_blocks == 0 {
-        return Ok(());
+        return Ok(()); // FIXME: we want it to fire in this case with empty
```

Comment:
⚠️ I believe this is the case where we want it to fire with empty accounts - need to figure out how to make that actually happen

Reply:
I think this is just a spot where the local ledger is empty and so there is nothing to do.